content/2-7.md

```diff
@@ -130,66 +135,69 @@ Optional ideas to support assessment:
-* Demonstrate accessible and inaccessible modal windows. Emphasize how focus remains within the boundaries of the relevant elements of such dialogs when they have been coded appropriately. Explain that specific focus treatment should be applied to the contents that are not part of the modal window as long as it is displayed so that they are not reachable using the keyboard
+* Demonstrate assistive technology interaction with rich applications such as Single Page Applications (SPA). Explain that assistive technologies may not perceive content changes by default unless there is a page reload. Emphasize that each time the view changes, a message indicating the currently selected view should be provided to assistive technologies via a live region.
+* Explain techniques to set focus to the most relevant place in an application, such as a newly loaded screen, the default button in a dialog, or the control that originated a modal dialog. Mention that setting focus to the most relevant place in a rich application is a shared responsibility among different team members: designers and developers.
+* Demonstrate the use of mechanisms to obtain information about the available keyboard shortcuts in rich applications, such as web-based email clients, spreadsheets, and word processors. Explain that when an application makes extensive use of non-standard keyboard shortcuts, a mechanism that allows users to obtain information about these shortcuts is necessary. Mention that providing such messages is a shared responsibility among different team members: designers and developers.
+* Demonstrate accessible and inaccessible modal dialogs. Emphasize how focus remains within the boundaries of the dialogs when they have been coded appropriately. Explain that specific focus treatment should be applied to the contents that are not part of the modal dialog as long as it is displayed so that they are not reachable using the keyboard
 
 #### Ideas to Assess Knowledge for Topic
 
 Optional ideas to support assessment:
 
-* Practical &mdash; Students are presented with an application with several views that show or hide based on user interaction and are asked to set focus to the specific displayed widget. Assess students' knowledge of how to place focus on a specific application widget based on its state or relevance.
+* Practical &mdash; Students are presented with a Single Page Application (SPA) with several screens that show or hide based on user interaction. The students are asked to set focus to the specific displayed widget. Assess students' knowledge of how to place focus on a specific application widget based on its state or relevance.
```

Comment:
```suggestion
* Practical &mdash; Students are presented with a Single Page Application (SPA) with several screens that show or hide based on user interaction. Students are asked to set focus to the specific displayed widget. Assess students' knowledge of how to place focus on a specific application widget based on its state or relevance.
```

I was using "Students" instead of "The students in other assessment ideas throughout the curricula.